boards/nxp/imx95_evk/imx95_evk_mimx9596_a55_smp.dts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx95_a55.dtsi>
+#include "imx95_evk-pinctrl.dtsi"
+
+/ {
+	model = "NXP i.MX95 A55";
+	compatible = "fsl,mimx95";
+
+	chosen {
+		zephyr,console = &lpuart1;
+		zephyr,shell-uart = &lpuart1;
+		zephyr,sram = &sram0;
+	};
+
+	psci {
+		compatible = "arm,psci-0.2";
```

Comment:
Sorry for late comments, it should be psci v1.1, please fix, thanks.

Reply:
will update the psci version. Thanks!